translate/expression/py_expression.py

```diff
@@ -26,10 +26,10 @@ def call(node):
 """
 takes ast.name node from python ast and converts to string 
 represenation for the generic ast
+FIXME: this should prob be in helpers since it is also used by assign
 """
 def name(node):
     if node.id == "print":
         return "logStatement"
     else:
-        return "customStatement"
-      
\ No newline at end of file
+        return node.id      
```

Comment:
Out of curiosity, what's the fix here?

Reply:
I was storing customStatement instead of the actual name. It was a remnant of how we were planning on dealing with custom functions that I forgot to remove. This is another reason why we are going to need types in our gast